packages/contracts-bedrock/src/cannon/MIPS64.sol

```diff
@@ -63,8 +63,8 @@ contract MIPS64 is ISemver {
     }
 
     /// @notice The semantic version of the MIPS64 contract.
-    /// @custom:semver 1.0.0
-    string public constant version = "1.0.0";
+    /// @custom:semver 1.1.0-beta.1
+    string public constant version = "1.1.0-beta.1";
```

Comment:
Not sure what the version should be here - are we moving to a different versioning process?

Reply:
cc @maurelian

Reply:
You can keep this as 1.0.0 since this hasn't been deployed to production yet.

Reply:
We need to update the guidance, but for now: 
1. keep the beta tag, I'd make it `1.0.0-beta.1` since it's never been released into production.
2. plan to remove it before deploying to production.